edb/lib/ext/auth.edgeql

```diff
@@ -17,4 +17,24 @@
 #
 
 
-CREATE EXTENSION PACKAGE auth VERSION '1.0';
+CREATE EXTENSION PACKAGE auth VERSION '1.0' {
+    set ext_module := "ext::auth";
+
+    create module ext::auth;
+
+    create type ext::auth::Identity {
+        create required property iss: std::str;
```

Comment:
Yeah, sure - even if it's just that we happened to borrow the concept from OpenID, `iss` sounds like a general-enough term.

Oh, does this mean, `iss` would be a URL as in OIDC, instead of a code name/identifier?

Reply:
> Oh, does this mean, iss would be a URL as in OIDC, instead of a code name/identifier?

Oh, great question. I wasn't really intending for it to be a URL, but that kinda makes sense especially if we implement some kind of generic provider. Any drawbacks? I wonder what the issuer would be in the case of GitHub: `https://github.com` or `https://api.github.com`?

Reply:
The only drawback I can think of is a possible mismatch/confusion between our `iss` and the `iss` returned from the remote provider - perhaps we will add some special protocol (like `github+https://private.github.deploy.org`) when the `iss` returned from the provider is not quite enough for selecting a provider backend on our side. Then the user must be aware of such differences and not use the value in a wrong way.

Reply:
Alright, that seems fine if/when we get to a problem like that in the future. So for now, I'll add another property to the `BaseProvider` class for the provider's issuer URL 👍